test/e2e/main_test.go

```diff
@@ -112,6 +115,12 @@ func TestNodeStartup(t *testing.T) {
 	waitForPodsOrFail(spec, 1)
 }
 
+// Run some functions for testing rights not covered by integration tests
```

Comment:
Tests don't run in the `infinispan-operator` account, only the operator does.
We need another kubernetes object to verify roles directly from test.
I'm adding a commit with a fix

`testKube.Kubernetes` runs in the caller context (`kind-kind` in CI), to test the `infinispan-operator` role from we need another kubernetes object.
Let me add a commit

Reply:
Thanks @rigazilla . Looks very good.